ownership-chain/runtime/src/precompiles/test.rs

```diff
@@ -23,8 +22,5 @@ fn ethereum_precompiled_addresses_are_precompile() {
 	assert!(is_precompile(hash(3)).unwrap());
 	assert!(is_precompile(hash(4)).unwrap());
 	assert!(is_precompile(hash(5)).unwrap());
-	assert!(is_precompile(hash(1026)).unwrap());
```

Comment:
I have the suspect that this test will not fail...

Reply:
Added check for hash(1026) not being a precompile